src/content/docs/style-guide/images/image-annotations.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Best practices for using image annotations
+contentType: page
+template: basicDoc
+topics:
+  - Basic style guide
+  - Writing guidelines
+---
+
+import styleguideExampleCalloutBox from 'images/style-guide_screenshot-full_example_callout_box.png'
+
+import apisanddataNerdGraphDashboardEntity from 'images/apis-and-data_screenshot-full_NerdGraph-dashboard-entity.png'
+
+import styleguideExampleArrow from 'images/style-guide_screenshot-full_example_arrow.png'
+
+Image annotations are text and shapes in screen captures or diagrams that help to highlight information and focus attention on the most important aspects of the screenshot. Avoid the use of text inside a screenshot, use [captions](/docs/style-guide/images/screenshots-images/#captions) instead.
```

Comment:
I would put "To" at the beginning: "To avoid the use of text inside a screenshot, use [captions](/docs/style-guide/images/screenshots-images/#captions) instead.

Reply:
ok, I'm going to add it. Thanks!!